### PR TITLE
Updating VS Extension Tools

### PR DESCRIPTION
**Pull request by maliha-srbd**
### Change Description ###

Uploading this PR on behalf of the SRBD Tizen VS Extension team.

### CODE REVIEW

File: docs/application/vstools/tools/install-tizen-app.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# To Install Tizen Applications via "Install Tizen App" Button
+The **Install Tizen App** feature streamlines the process of building and deploying Tizen Applications directly from your development environment. Whether you're working on Tizen .NET Application, Web Application or Native Application, this feature simplifies the installation to selected Emulator with just a click. It supports the installation of .tpk, .rpk, and .wgt package formats.
+## Prerequisites
+Before using the "Install Tizen App" feature, ensure that you have the following setup:
+* **Visual Studio Tools for Tizen installation:** Make sure you have the latest version of [Visual Studio Tools for Tizen](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsTizen2022) installed on your machine.
```
Comment (safir-srbd) [inline docs/application/vstools/tools/install-tizen-app.md:5]:
```suggestion
* **Visual Studio Tools for Tizen installation:** Make sure you have the latest version of [Visual Studio Tools for Tizen](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsTizen2022){:target="_blank"} installed on your machine.
```

File: docs/application/vstools/tools/install-tizen-app.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* **Configured Emulator:** An Emulator should be selected before you click the **Install Tizen App** button. If no Emulator is selected, the [Emulator Manager](https://docs.tizen.org/application/vstools/tools/emulator-manager/) will open, and prompt you to select an Emulator.
```
Comment (safir-srbd) [inline docs/application/vstools/tools/install-tizen-app.md:6]:
```suggestion
* **Configured Emulator:** An Emulator should be selected before you click the **Install Tizen App** button. If no Emulator is selected, the [Emulator Manager](./emulator-manager.md) will open, and prompt you to select an Emulator.
```

File: docs/application/vstools/tools/install-tizen-app.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    For Tizen .NET application, it should include a valid `tizen-manifest.xml` file.
+## Step-by-step guide
+Follow these steps to build and install your Tizen Application using the **Install Tizen App** button:
+
+1. Right-Click on the **Tizen Project**:
+<br>In the **Solution Explorer** panel of your Visual Studio IDE, locate your Tizen project. Right-click on the project name to open the context menu.
+![Tizen Project Right Click](./media/install-tizen-project-right-click.png)
+
+2. Select the **Install Tizen App** button:
+<br>From the context menu, select the **Install Tizen App** option. This action initiates the build and deployment process. The last used Emulator will be chosen automatically. If no Emulator is selected, the [Emulator Manager](https://docs.tizen.org/application/vstools/tools/emulator-manager/) will open, prompting you to select an Emulator.
```
Comment (safir-srbd) [inline docs/application/vstools/tools/install-tizen-app.md:17]:
```suggestion
<br>From the context menu, select the **Install Tizen App** option. This action initiates the build and deployment process. The last used Emulator will be chosen automatically. If no Emulator is selected, the [Emulator Manager](./emulator-manager.md) will open, prompting you to select an Emulator.
```

File: docs/application/vstools/tools/leak-sanitizer.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    
+      <img src="./media/LSan Menu.png" alt="LSAN Menu" width="980"/>
+
+2. After the application launches, run the application scenario that you want to test. Unlike the Address Sanitizer, Visual Studio Code checks and shows the profiling result when you exit the application. As illustrated in the following figure:
+  
+      <img src="./media/Lsan Stop profiling.PNG" alt="LSAN stop" width="980"/>
+
+    After clicking the **Stop Profiling** button, VS Code will collect and show the profiling report. As illustrated in the following figure:
+  
+      <img src="./media/LSan report.PNG" alt="LSAN Report" width="980"/>
```
Comment (safir-srbd) [inline docs/application/vstools/tools/leak-sanitizer.md:24]:
```suggestion
1. In the Project Explorer view, select **Tools >  Tizen > Profiler > Profile With Leak Sanitizer**. 
  
   The profiler links your program to a runtime library containing the bare necessities required for Leak Sanitizer to work, this is done by setting the option `-fsanitize` to `leak`, and no compile-time instrumentation is applied here. Whilst the option is set, the application launches on the connected target device or the Emulator to detect any runtime memory leaks. As illustrated in the following figure:
    
   ![Leak Sanitizer Menu](./media/LSan Menu.png)

2. After the application launches, run the application scenario that you want to test. Unlike the Address Sanitizer, Visual Studio Code checks and shows the profiling result when you exit the application. As illustrated in the following figure:
  
   ![Leak Sanitizer Stop](./media/Lsan Stop profiling.PNG)

   After clicking the **Stop Profiling** button, VS Code will collect and show the profiling report. As illustrated in the following figure:
  
   ![Leak Sanitizer Report](./media/LSan report.PNG)
```

File: docs/application/tizen-studio/common-tools/emulator-manager.md [hunk 5]
```diff
@@ -183,6 +183,21 @@ The following figure shows an example of how to create and launch an emulator us
...
+2. Click on the **Issue Reporter** button to be redirected to a **GitHub** page that will open in your default browser. Then use your GitHub username or email address, and password to Sign in:
+
+    ![Sign In Page](./media/sign_in.png)
+3. After signing in, a new page will open. It can be used to report two types of bugs/issues:
+
+    a. **Documentation bug report** - Click on the **“Get started”** button (red rectangle), to report any issue associated with the documentation of Emulator Manager.
+
+    b. **Platform bug report** - Click on the **“Get started”** button (blue rectangle) below, to report any issue that arises while using Emulator Manager.
+
+    ![GitHub Page](./media/git_page.png)
```
Comment (safir-srbd) [inline docs/application/tizen-studio/common-tools/emulator-manager.md:199]:
```suggestion
    ![GitHub Page](./media/git_page.PNG)
```

File: docs/application/tizen-studio/common-tools/emulator-manager.md [hunk 6]
```diff
@@ -183,6 +183,21 @@ The following figure shows an example of how to create and launch an emulator us
 
 In the `create` command, the `–p` option is the platform name. You can see the available platforms using the `list -vm` command.
 
+## Report an issue in Emulator Manager
+The **Issue Reporter** button can be used to report any issue that can arise while using **Emulator Manager**. This button is placed inside the main window of **Emulator Manager** and makes the process of reporting an issue straightforward. To report an issue, follow these steps:
+1. Launch **Emulator Manager** and view the top right region where two buttons appear: **Issue Reporter** (red rectangle), and **About** buttons are found, as illustrated in the following figure:
+![Emulator Manager Window](./media/em_manager.png)
```
Comment (safir-srbd) [inline docs/application/tizen-studio/common-tools/emulator-manager.md:189]:
```suggestion
![Emulator Manager Window](./media/em_manager.PNG)
```

File: docs/application/vstools/tools/unit-test-with-code-coverage.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+    * Upon double click on the failed test case, the cursor will navigate to the line the test case failed. As illustrated in the following figure:
+      
+      <img src="./media/utc_12.png" alt="HTML Code Coverage Report 1" width="980"/>
+
+      <!-- ![Test results](./media/utc_12.png) -->
+
+ * ### HTML Code Coverage report
+    * The **index.html** page is opened in the browser to view the Code Coverage Report, as illustrated in the following figure:
+      
+      <img src="./media/coverage_report_1.png" alt="HTML Code Coverage Report 1" width="880"/>
```
Comment (safir-srbd) [inline docs/application/vstools/tools/unit-test-with-code-coverage.md:110]:
```suggestion
      ![HTML Code Coverage Report](./media/coverage_report_1.PNG)
```

File: docs/application/vstools/tools/unit-test-with-code-coverage.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+      <!-- ![Test results](./media/coverage_report_1.png) -->
+
+    * Click on any of the projects to get the coverage at the file level, as illustrated in the following figure:
+
+      <img src="./media/coverage_report_2.png" alt="HTML Code Coverage Report 1" width="880"/>
```
Comment (safir-srbd) [inline docs/application/vstools/tools/unit-test-with-code-coverage.md:116]:
```suggestion
      ![HTML Code Coverage Report](./media/coverage_report_2.PNG)
```

File: docs/application/vstools/tools/unit-test-with-code-coverage.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+The Code Coverage tool is based on the `llvm-cov` tool. Using the Code Coverage tool, you can detect parts of code that are not covered by a unit test, and ensure that the uncovered code does not cause errors at runtime. The Code Coverage tool supports functional, statement, and branch coverage.
+
+<!-- ## Use Unit Test and Code Coverage -->
+> [!NOTE]  
+> Before you run the Unit Test and Code Coverage, follow the steps below:
+> - Make sure that you have an Emulator or a connected target device running.
+> - If you want to try out the tool and do not have an applicable project to test, create a project with the **Project Wizard** using a template or a sample. For more information on creating a project, check [Native Application Development](https://docs.tizen.org/application/vstools/Tizen/native).
```
Comment (safir-srbd) [inline docs/application/vstools/tools/unit-test-with-code-coverage.md:11]:
```suggestion
> - If you want to try out the tool and do not have an applicable project to test, create a project with the **Project Wizard** using a template or a sample. For more information on creating a project, check [Native Application Development](../Tizen/native).
```

**Review by safir-srbd — COMMENTED** [review]
@maliha-srbd  Please commit the changes.

File: docs/application/vstools/tools/unit-test-with-code-coverage.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+        <!-- ![Test results](./media/utc_10.png) -->
+
+      The method declaration should be mentioned in the header file.
+
+        <img src="./media/utc_8.png" alt="method mention in header file" width="480"/>
+
+        <!-- ![Test results](./media/utc_8.png) -->
+
+    * Enclose the main() function of the Native project with `TEST_BUILD`:
+      ```cpp
```
Comment (RF-Faisal) [inline docs/application/vstools/tools/unit-test-with-code-coverage.md:86]: The code section is highlighted in the github preview but not in the website. C++ highlighter is supposed to work but not sure why it is not working here. Please try replacing line 86 by:
```csharp

File: docs/application/vstools/Tizen/native.md [hunk 11]
```diff
@@ -52,3 +52,13 @@ To create a Tizen Native project:
    ![Add break point](media/native_debug_application.PNG)
 
 3. Start the debugging session by selecting **Debug &gt; Start Debugging** in the menu bar, or pressing **F5**, or by clicking the **Debug** button in the menu bar.
+
```
Comment (RF-Faisal) [inline docs/application/vstools/Tizen/native.md:55]: The whole list is not highlighted. Only the first point is highlighted out of the three.

**Review by safir-srbd — APPROVED** [review]
@maliha-srbd LGTM

**Pull request merged**